charts/shoot-core/components/charts/apiserver-proxy/templates/psp/apiserver-proxy-psp.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: {{ include "podsecuritypolicyversion" .}}
+kind: PodSecurityPolicy
+metadata:
+  name: gardener.kube-system.apiserver-proxy
+spec:
+  privileged: true
+  volumes:
+  - hostPath
```

Comment:
How `hostPath` is working with `   allowedHostPaths: [] `?

Reply:
`hostPath` must be removed.